style.css

```diff
@@ -50,39 +50,39 @@ h2 {		     /* This styles the headers within the page */
 div.section {
 	padding: 1em;
 	margin-bottom: 0;
-	text-align: justify;
+	text-align: center;
 
 }
 
 /* If we want the divs with ids plan and more to look the same, we can put them together */
 #plan, #more {
-	background:#000080;
+	background:#800000;
 	color: white;
-	font: 'Source Sans Pro',Arial, Sans-Serif;
+	font: 'Source Sans Pro',New Times Roman, Sans-Serif;

```

Comment:
This says `New Times Roman` but I think it is called `Times New Roman`. Can you change that to `Times New Roman`?